extensions/amp-lightbox/1.0/component.jss.js

```diff
@@ -38,39 +38,61 @@ before allowing buttons to be activated. */
 };
 
 const wrapper = {
-  top: 0,
-  bottom: 0,
-  left: 0,
-  right: 0,
-  width: '100%',
-  height: '100%',
-  position: 'fixed',
-  boxSizing: 'border-box',
-};
+  top: '0 !important',
+  left: '0 !important',
+  bottom: '0 !important',
+  right: '0 !important',
+  width: '100% !important',
+  height: '100% !important',
+  position: 'fixed !important',
+  boxSizing: 'border-box !important',
+
+  /** Scroll containment:
+   *  overflow: hidden does not trigger overscrollBehavior, so
+   *  overflow: scroll is applied even though the lightbox should not scroll.
+   *  We rely instead on the overflow: hidden in content layer to prevent
+   *  scrolling inside the lightbox.
+   */
+  overflow: 'scroll !important', // Prevent scrolling inside lightbox.
```

Comment:
Do you mean "outside" lightbox?

Reply:
Yes, removed the comment on this line and below as it's covered in the larger comment section above this.